platform-configs/include/protectli-vp66xx.robot

```diff
@@ -0,0 +1,76 @@
+*** Settings ***
+Resource    protectli-pro.robot
+Resource    protectli-common.robot
+
+
+*** Variables ***
+${FLASH_SIZE}=                      ${16*1024*1024}
+
+${DEVICE_AUDIO1}=                   ALC897
+${DEVICE_AUDIO2}=                   Elkhartlake HDMI
+${DEVICE_AUDIO1_WIN}=               High Definition Audio Device
+${INITIAL_CPU_FREQUENCY}=           2600
+${MAX_CPU_TEMP}=                    82
+${WATCHDOG_SUPPORT}=                ${TRUE}
+
+# eMMC driver support
+${E_MMC_NAME}=                      AJTD4R
+
+${DMIDECODE_FIRMWARE_VERSION}=      Dasharo (coreboot+UEFI) v0.9.0
+${DMIDECODE_RELEASE_DATE}=          02/19/2024
+
+
+*** Keywords ***
+Power On
+    RteCtrl Power Off    ${6}
+    Sleep    5s
+    Power Cycle On
+
+Flash Device Via External Programmer
+    [Documentation]    Flash Device Under Test firmware, check flashing result
+    ...    and set RTE relay to ON state. Implementation must be
+    ...    compatible with the theory of operation of a specific
+    ...    platform.
+    Sonoff Power On
```

Comment:
I wonder if we can start replacing these flashing keywords with the code from osfv_cli. Support for this board is in: https://github.com/Dasharo/osfv-scripts/pull/25

Reply:
This may require a bit more work. Since what we have works, let's continue with what we have. Some WIP work: https://github.com/Dasharo/open-source-firmware-validation/issues/257